vision/classification/leafsnap30/README.md

```diff
@@ -122,7 +122,11 @@ https://github.com/microsoft/onnxruntime/blob/master/tools/python/PythonTools.md
 If this PR does update/add .onnx or .tar.gz files, please use `python workflow_scripts/generate_onnx_hub_manifest.py --target diff` to update ONNX_HUB_MANIFEST.json with according model information (especially SHA) for ONNX Hub.
 
 ### References
-Link to paper or references.
+[Leafsnap: A Computer Vision System for Automatic Plant Species Identification](https://rdcu.be/c0aBX) (original LeafSnap paper)
```

Comment:
You could consider adding a year of publication after the title, or authors for that matter.